### PR TITLE
Migration guide for new assistant

### PR DESCRIPTION
**Pull request by bethaitman**
Have not added release note yet.

I'd particularly appreciate:
- suggestions for the start of the doc, explaining the good things about the new assistant
- a thorough check of the tables, to make sure I've got the migration details correct

### CODE REVIEW

File: docs/source/support/new-assistant.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Moving to the new DAML assistant
+################################
+
+New command-line tool for working with the DAML SDK: DAML Assistant, or ``daml``. Many commands are similar to using the old SDK Assistant (``da``), but with some changes:
+
+- Simplified installation process
+- Consistent behaviour of the ``new`` command
```
Comment (cocreature) [inline docs/source/support/new-assistant.rst]: What exactly does “consistent” mean here?

File: docs/source/support/new-assistant.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+- In-built templates mechanism mostly gone, replaced with more standard mechanism of cloning from open-source repositories
+- Components don't run in the background, so you stop them with ``ctrl+c``
```
Comment (cocreature) [inline docs/source/support/new-assistant.rst]: Maybe mention `daml start` explicitly here?

File: docs/source/support/new-assistant.rst (hunk 1 above)
Comment (bethaitman) [reply docs/source/support/new-assistant.rst]: Yeah, this is a bit first-draft. Let me find what Fran's original notes said:

> The `new` command now takes arguments in a consistent order

I think that's better

File: docs/source/support/new-assistant.rst (hunk 2 above)
Comment (bethaitman) [reply docs/source/support/new-assistant.rst]: Agreed, good shout

**Comment by bethaitman** [comment]
I've made my final changes: I now consider this done enough for a first go (in that, this is as good as it'll get without user feedback, so that's what we should look for next.).

**Pull request merged**